start.go

```diff
@@ -61,8 +67,12 @@ func start(c *cli.Context) {
 	templateApis := template.GetAllAWSServerlessApiResources()
 
 	for name, api := range templateApis {
-		log.Printf("Adding new API: %s", name)
-		mux.AddAPI(&api)
+		successMsg.Printf("Adding new API: %s\n\n", name)
```

Comment:
There is already a message somewhere that prints the mountpoints. Can we make this a debug log instead?

Reply:
Yes, I'll update the PR.